### ISSUE TITLE
Long-term stress testing of Codewind 

### ISSUE THREAD

**Issue by johnmcollier** [eclipse/codewind#1164]
We're frequently testing Codewind with new / fresh installs, and aren't doing much long-term testing (or stress testing for that matter) on older installs.

We need to be doing long-term stress testing of Codewind to see how well it holds up over time (and with heavy load over time). A month-old install (or even longer) of Codewind should be just as functional as a fresh install.

**Comment by jgwest** [comment]
A great way to do stress testing like this is to keep running your existing automated tests over and over again, against a single PFE instance/cluster, without restarting it, until a failure occurs (or if failures occur too often, then it run X number of times).

**Comment by tobespc** [comment]
How does this tie in with SVT testing work?   They might have something already

**Comment by ssh24** [comment]
This is definetely do-able. We can test previous Codewind releases but we also need to make sure our test source is also pointing to the same release tag.

**Comment by johnmcollier** [comment]
@jagraj Is this something your test squad would be able to look at and / or help with at some point?

**Comment by jagraj** [comment]
@johnmcollier 
Yes, we are going to work on this once in future releases. Now we have selenium integration tests working for Che and we need to work on load testing with multiple che users. I also see Eclipse Che using selenium as their automation test framework and I need to investigate how they are running their tests for multi users feature.

**Comment by ssh24** [comment]
This is nice to have on org-wide integration tests but does not stop tribe based tests to be run based on release flags as well. For example, turbine and portal can also enhance their tests so it can be ran based on an older release.

**Comment by elsony** [comment]
@ssh24 I think those are two separate items.  I agree there should be an easy way for us to be able to run our test on specific release.  However, for the performance/stress testing, those are typical SVT level of testing and if SVT already have plan for it, there is no need to duplicate the effort.  Maybe @jagraj can share the scenario that they planned to cover and we can review that to see if there is any scenario that we find is not covered and request SVT to add those.

**Comment by ssh24** [comment]
That sounds good to me.

**Comment by rwalle61** [comment]
linking this to testing epic https://github.com/eclipse/codewind/issues/2291
